CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/ImportTableColumn.java

```diff
@@ -85,7 +86,8 @@ public boolean validate(final List<TableRow> data) {
                         : BooleanAttributeDescription.class.getDeclaredConstructor().newInstance();
 
                 for (final TableRow row : data) {
-                    columnFailed = processRow(row, attributeDescription, parser, parserParameters, defaultValue);
+                    prevColumnFailed = columnFailed;
+                    columnFailed = processRow(row, attributeDescription, parser, parserParameters, defaultValue) || prevColumnFailed;
```

Comment:
could these be simplified to `columnFailed = processRow(...) || columnFailed`? Would remove the need for `prevColumnFailed`